resources/server/marketplace/marketplace.service.ts

```diff
@@ -126,10 +126,10 @@ class _MarketplaceService {
       if (reportExists) {
         marketplaceLogger.error(`This listing has already been reported`);
         resp({ status: 'error', errorMsg: 'REPORT_EXISTS' });
```

Comment:
Instead of adding that else statement we can also just add `return` keyword here.

Reply:
whatever you prefer 🤷🏻